cubedash/_utils.py

```diff
@@ -44,6 +46,10 @@
 
 _LOG = structlog.get_logger()
 
+def render(template, **context):
+    theme = flask.current_app.config.get('THEME', flask.current_app.config['DEFAULT_THEME'])
```

Comment:
I don't think we need both properties (we're already using [setdefault() above](https://github.com/opendatacube/dea-dashboard/pull/46/files#diff-456fe973ee5300fc13637fe6e05c6e11R34))

I lean towards replacing it with one "CUBEDASH_THEME" property.